packages/CodeGen/config.json

```diff
@@ -188,10 +188,18 @@
             "type": "CoreEntitySubclasses",
             "directory": "../MJCoreEntities/src/generated"
         },
+        {
```

Comment:
@JS-BC this isn't really needed. If you're going to output these new definitions to the same place as the entities, just use those to remove the complexity you've added here